src/riak_kv_entropy_manager.erl

```diff
@@ -819,19 +802,7 @@ get_aae_throttle_limits() ->
 %% List sorting is not required: the throttle is robust with any ordering.
 
 set_aae_throttle_limits(Limits) ->
-    case {lists:keyfind(-1, 1, Limits),
-          catch lists:all(fun({Min, Lim}) ->
-                                  is_integer(Min) andalso is_integer(Lim) andalso
-                                      Lim >= 0
-                          end, Limits)} of
-        {{-1, _}, true} ->
-            lager:info("Setting AAE throttle limits: ~p\n", [Limits]),
-            application:set_env(riak_kv, aae_throttle_limits, Limits),
-            ok;
-        {Else1, Else2} ->
-            {error, [{negative_one_length_is_present, Else1},
-                     {all_sleep_times_are_non_negative_integers, Else2}]}
-    end.
+    riak_core_throttle:set_limits(?AAE_THROTTLE_ENV_KEY, Limits).
 
 query_and_set_aae_throttle(#state{last_throttle=LastThrottle} = State) ->
     case get_aae_throttle_kill() of
```

Comment:
When we invert the kill/enabled thing, need to change this function name and just use the `enabled` value - switch the case statement around, etc.